app/Platform/Concerns/BuildsGameListQueries.php

```diff
@@ -151,6 +151,26 @@ private function applyFilters(Builder $query, array $filters, ?User $user = null
                 continue;
             }
 
+            /*
+             * only show games based on their tags
+             */
+            if ($filterKey === 'game-type' && !empty($filterValues)) {
+                if (in_array('retail', $filterValues)) {
+                    // Only return untagged games.
+                    $query->whereNotExists(function ($subquery) {
+                        $subquery->select(DB::raw(1))
+                            ->from('tags')
+                            ->join('taggables', 'tags.id', '=', 'taggables.tag_id')
+                            ->whereColumn('taggables.taggable_id', 'GameData.ID')
+                            ->where('taggables.taggable_type', 'game')
+                            ->where('tags.type', 'game');
+                    });
+                } else {
+                    $query->withAnyTags($filterValues, 'game');
```

Comment:
This results in something extracting the english json tags from the tags table. Does it work in other languages? As far as I can tell, the json in the table only has english entries.

![image](https://github.com/user-attachments/assets/a34a24c9-4d42-4eb5-b985-bf0a87788b00)

Reply:
This is an unfortunate quirk of `spatie/laravel-tags`. There was some prior discussion in #2963:

> Another thing to keep in mind is spatie/laravel-tags localizes tags, and there is no way to disable this behavior:
> We should probably only store "en" values in the database, and let the React UI do the presentation/localization work.

I've confirmed other locales can use this filter without any issues.